From f00201fa6ca92de3e1c63fb499ea33f610ea02b9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Thu, 12 Sep 2019 14:50:22 +0200
Subject: [PATCH 090/152] ci: Disable MR handling for now

We aren't quite ready to enforce non-legacy style for now, mostly due
to the xgettext bug that prevents us from using template strings in
some places, and vast amounts of legacy indentation.

https://gitlab.gnome.org/GNOME/gnome-shell/merge_requests/716
---
 .gitlab-ci/run-eslint.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/.gitlab-ci/run-eslint.sh b/.gitlab-ci/run-eslint.sh
index 8466d97c3..ae4aee979 100755
--- a/.gitlab-ci/run-eslint.sh
+++ b/.gitlab-ci/run-eslint.sh
@@ -66,6 +66,10 @@ create_common() {
   rm $OUTPUT_FINAL.tmp
 }
 
+# Disable MR handling for now. We aren't ready to enforce
+# non-legacy style just yet ...
+unset CI_MERGE_REQUEST_TARGET_BRANCH_NAME
+
 if [ "$CI_MERGE_REQUEST_TARGET_BRANCH_NAME" ]; then
   git fetch $CI_MERGE_REQUEST_PROJECT_URL.git $CI_MERGE_REQUEST_TARGET_BRANCH_NAME
   branch_point=$(git merge-base HEAD FETCH_HEAD)
-- 
2.23.0

